functest/main_test_wrapper.go

```diff
@@ -246,15 +273,22 @@ func waitForNet() error {
 }
 
 func startNet() error {
+
 	cwd, err := os.Getwd()
 	if err != nil {
 		return errors.Wrap(err, "[ startNet ] Can't get current working directory")
 	}
 	defer os.Chdir(cwd)
 
-	err = os.Chdir("../")
-	if err != nil {
-		return errors.Wrap(err, "[ startNet  ] Can't change dir")
+	for cwd[len(cwd)-15:] != "insolar/insolar" {
```

Comment:
it wouldn't work with go modules, because project's base/working dir could be in path without `insolar/insolar`

Reply:
at the moment this change is necessary, agreed after discussion by @nordicdyno